src/Model/Generated/Card.php

```diff
@@ -158,11 +158,19 @@ class Card extends BunqModel
     /**
      * Array of Types, PINs, account IDs assigned to the card.
      *
-     * @var CardPinAssignment
+     * @var CardPinAssignment[]
      */
     protected $pinCodeAssignment;
 
     /**
+     * ID of the MA to be used as fallback for this card if insufficient
```

Comment:
This is not generated right ? 😁 Please spell out monetary account

Reply:
@OGKevin What makes you think it is not generated? `src/Model/Generated/Card.php`

Reply:
Aaaahhhhh .gitattributes, will make PR tonight.